admin_scripts/reposync/reposync.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+pushd $(dirname "$0")
+source ./reposync.config
+popd
```

Comment:
Replace with `source $BASE/conf/reposync.config`

Reply:
Oh I think I see now.

So set it up similar to mk_new_node.sh where it has
```
BASE=___INSTALL_DIR___
source $BASE/conf/reposync.config
```
I was looking for something that search and replaced BASE, but I see it instead has something to search and replace ___INSTALL_DIR___

Reply:
Yep, that `BASE=___INSTALL_DIR___` line is the key. The triple underscore part get's replaced with the actual path when `setup.sh` runs.

Reply:
The change for setting `BASE=___INSTALL_DIR___` has been made.